package.json

```diff
@@ -90,5 +95,8 @@
     "vuetify-loader": "^1.9.2",
     "vuex-class": "^0.3.2",
     "vuex-module-decorators": "^1.2.0"
+  },
+  "overrides": {
+    "webpack": "5.78"
```

Comment:
This needed to be included, if it wasn't - it would run two versions of webpack (vue-cli-service would target the old one)